loan-payment-skip/REPWRITERSPECS/BANNO.LOANPAYMENT.SKIP.V1.POW

```diff
@@ -454,14 +469,17 @@ PROCEDURE PROCPERFORMSKIPAPAYMENT
          NEWLINE
          PRINT Q+"nextDueDate"+Q+": "+Q+FORMAT("99/99/9999",LNEWDUEDATE)+Q
          NEWLINE
-         FOR EACH SHARE WITH (SHARE:ID=@RGUSERCHR2)
+	       FOR EACH SHARE WITH (SHARE:ID=@RGUSERCHR2)

```

Comment:
Don't mix tabs and spaces pls.

Reply:
corrected - thanks

Reply:
searched for and found a handful more.  fixed